setup.py

```diff
@@ -248,7 +248,7 @@ def run(self):
     'setuptools >=42',
     'requests',
     'packaging',
-    'bokeh ==3.4.0rc1',
+    'bokeh >=3.4.0rc2,<3.5.0',
```

Comment:
I think this will pull in Bokeh 3.4.0.dev8

Reply:
It did 
![image](https://github.com/holoviz/panel/assets/19758978/7e559fe0-05cf-419b-8846-e7af7a5820c2)

Reply:
On conda?

Reply:
😢

Reply:
Yes, because of conda.